### PR TITLE
Change header of homepage to h1

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
With the work I've done by removing the SCSS from the app, I've changed most of the headers already to a h1. 
The homepage had a `visibilityHidden`  h1 that yielded the same that is in the `topicalHeader` so that is by default a h1 now and removed the unnecessary hidden text, that was not even working.

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Review by VWSCoronaDashboard10 — APPROVED** [review]

File: packages/app/src/domain/topical/topical-section-header.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export function TopicalSectionHeader({
           alignItems="baseline"
         >
           <Heading
-            level={2}
+            level={1}
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/domain/topical/topical-section-header.tsx:58]: I don't get this. The component is called a section header. How is it ok to assume that every section will be H1? Seems like it's by definition not a section then.

Comment (VWSCoronaDashboard14) [reply packages/app/src/domain/topical/topical-section-header.tsx:58]: In my opinion, all those headings should be an h1 in the way they are used. Following your feedback, I rather rename the component `topical-header` or something similar. I don't expect them they get used more than once per page. @VWSCoronaDashboard8

Comment (VWSCoronaDashboard8) [reply packages/app/src/domain/topical/topical-section-header.tsx:58]: There seem to be 3 pages that each use 5 `TopicalSectionHeader` components, so I don't see how that would work?

Comment (VWSCoronaDashboard14) [reply packages/app/src/domain/topical/topical-section-header.tsx:58]: My bad, didn't look good 😢

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Pull request merged**